lib/plaid/identity_verification.ex

```diff
@@ -0,0 +1,589 @@
+            created_at: nil,
+            completed_at: nil,
+            previous_attempt_id: nil,
+            shareable_url: nil,
+            status: nil,
+            template: nil,
+            user: nil,
+            steps: nil,
+            documentary_verification: nil,
+            selfie_check: nil,
+            kyc_check: nil,
+            risk_check: nil,
+            watchlist_screening: nil,
+            request_id: nil
+
+  @type t :: %__MODULE__{
+          id: String.t(),
+          client_user_id: String.t(),
+          created_at: String.t(),
+          completed_at: String.t() | nil,
+          previous_attempt_id: String.t() | nil,
+          shareable_url: String.t() | nil,
+          status: String.t(),
+          template: Plaid.IdentityVerification.Template.t(),
+          user: Plaid.IdentityVerification.User.t(),
+          steps: Plaid.IdentityVerification.Steps.t(),
+          documentary_verification: Plaid.IdentityVerification.DocumentaryVerification.t() | nil,
+          selfie_check: Plaid.IdentityVerification.SelfieCheck.t() | nil,
+          kyc_check: Plaid.IdentityVerification.KYCCheck.t() | nil,
+          risk_check: Plaid.IdentityVerification.RiskCheck.t() | nil,
+          watchlist_screening: Plaid.IdentityVerification.WatchlistScreening.t() | nil,
+          request_id: String.t()
+        }
+  @type params :: %{required(atom) => term}
+  @type config :: %{required(atom) => String.t() | keyword}
+  @type error :: {:error, Plaid.Error.t() | any()} | no_return
+
+  defmodule Template do
+    @moduledoc """
+    Plaid Identity Verification Template data structure.
+    """
+
+    @derive Jason.Encoder
+    defstruct id: nil, version: nil
+
+    @type t :: %__MODULE__{
+            id: String.t(),
+            version: integer()
+          }
+  end
+
+  defmodule User do
+    @moduledoc """
+    Plaid Identity Verification User data structure.
+    """
+
+    @derive Jason.Encoder
+    defstruct email_address: nil,
+              phone_number: nil,
+              date_of_birth: nil,
+              ip_address: nil,
+              name: nil,
+              address: nil,
+              id_number: nil
+
+    @type t :: %__MODULE__{
+            email_address: String.t() | nil,
+            phone_number: String.t() | nil,
+            date_of_birth: String.t() | nil,
+            ip_address: String.t() | nil,
+            name: Plaid.IdentityVerification.User.Name.t() | nil,
+            address: Plaid.IdentityVerification.User.Address.t() | nil,
+            id_number: Plaid.IdentityVerification.User.IDNumber.t() | nil
+          }
+
+    defmodule Name do
+      @moduledoc """
+      Plaid Identity Verification User Name data structure.
+      """
+
+      @derive Jason.Encoder
+      defstruct given_name: nil, family_name: nil
+
+      @type t :: %__MODULE__{
+              given_name: String.t(),
+              family_name: String.t()
+            }
+    end
+
+    defmodule Address do
```

Comment:
Is this meaningfully different from `Plaid.Accounts.Account.Owner.Address` which is just a map? Thoughts on standardizing address across the two modules?